include/SSVOpenHexagon/Components/CPlayer.hpp

```diff
@@ -34,6 +40,11 @@ class CPlayer
 
     bool dead;
     bool justSwapped;
+    bool forcedMove;
+
+    float radius;
+    float maxSafeDistance;
+    float currentSpeed;
```

Comment:
These fields should be documented as well, otherwise it will be hard to keep track of everything a few weeks from now :)